Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -321,6 +323,28 @@ public class ChatChannelListController: DataController, DelegateCallable, DataSt
         }
     }
 
+    func resetQuery(
```

Comment:
Should this be private? 🤔

Reply:
Nope, it is internal because we only want for it to be used inside our own module.

Reply:
👍🏼